### PR TITLE
Feat/message composer typing fix

### PR DESCRIPTION
**Pull request by artdnlv**
- feat(message): add typing for feed reply endpoint message attachment payload
- feat(message-template): add missing typing for interactive list in quick replies

https://app.clickup.com/t/24327122/TECH-5209

### CODE REVIEW

**Comment by eljefedelrodeodeljefe** [comment]
Task linked: [TECH-5209 (FE) Conversation View: Composer](https://app.clickup.com/t/24327122/TECH-5209)

File: src/entities/message-template/message-template.ts [hunk 1]
```diff
@@ -79,6 +79,38 @@ export interface MessageTemplateRawPayloadLocalizedContentItemAction {
...
+  replies?: MessageTemplateRawPayloadQuickReply[]
+}
+
+export interface MessageTemplateRawPayloadList {
+  body?: string
+  type?: 'list' | string | null
+  button?: {
+    text?: string | null
+    type?: string | null
+    button_type?: string | null
```
Comment (MaxAndDev) [inline src/entities/message-template/message-template.ts:95]:
```suggestion
    button_type?: string | null
    utm_source?: string | null
    utm_medium?: string | null
    utm_campaign? string | null
    utm_id?: string | null
    utm_term?: string | null
    utm_content?: string | null
```

Comment (MaxAndDev) [reply src/entities/message-template/message-template.ts:95]: this is probably not in the scope of that PR but we could add those here for the Link Manager as well ? wdyt @artdnlv

Comment (MaxAndDev) [reply src/entities/message-template/message-template.ts:95]: ah that the list type `MessageTemplateRawPayloadList` nvm

**Review by MaxAndDev — APPROVED** [review]

**Pull request merged**